db/migrate/20230327154626_add_confirmable_to_users.rb

```diff
@@ -0,0 +1,22 @@
+class AddConfirmableToUsers < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
+  # Note: You can't use change, as User.update_all will fail in the down migration
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
+    add_index :users, :confirmation_token, unique: true, algorithm: :concurrently
+    # User.reset_column_information # Need for some types of updates, but not for update_all.
+    # To avoid a short time window between running the migration and updating all existing
+    # users as confirmed, do the following
+    User.update_all confirmed_at: DateTime.now
```

Comment:
I think this should be moved into it's own AfterParty task.  This would overwrite every User's confirmed_at value if this migration is ever ran more than once.

The Post-deployment tasks section of the README has some examples on how to create a post deployment task

Reply:
I see, that makes sense. I rolled back the migration, edited the file, and moved this into its own after party task.